dm/dm/ctl/master/check_task.go

```diff
@@ -34,6 +34,7 @@ func NewCheckTaskCmd() *cobra.Command {
 	}
 	cmd.Flags().Int64P("error", "e", common.DefaultErrorCnt, "max count of errors to display")
 	cmd.Flags().Int64P("warn", "w", common.DefaultWarnCnt, "max count of warns to display")
+	cmd.Flags().String("start-time", "", "specify the start time of binlog replication, e.g. 2021-10-21 00:01:00")
```

Comment:
Need to remind users to pay attention to quotation marks

Reply:
https://github.com/pingcap/tiflow/pull/4261/commits/a0f9c7190f793ccb47e8dd6625b466a5d5cb20d9